pkg/kubectl/cmd/explain.go

```diff
@@ -106,19 +105,20 @@ func RunExplain(f cmdutil.Factory, out, cmdErr io.Writer, cmd *cobra.Command, ar
 	}
 
 	if len(apiVersionString) == 0 {
-		groupMeta, err := scheme.Registry.Group(gvk.Group)
-		if err != nil {
-			return err
+		if len(gvk.Version) == 0 {
```

Comment:
does this branch get hit?  I'm not seeing why we'd end up here.  The mapper should give us a specific version.

Reply:
It's not, it might be a leftover from the pre-mapper-handling-versions days ;) I'll remove it.